### ISSUE TITLE
I have used kappa metric provided here and getting a near zero values for complete disagreement instead of -1. Am i missing something?

### ISSUE THREAD

**Issue by prakashjayy** [benhamner/Metrics#33]
x = np.array([0,3,2,4,0,2,0,4,3,0,2])  
    y = np.array([2,1,3,2,3,4,2,1,4,3,1])
    print (kappa(x,y,min_rating = 0, max_rating = 4))

    [complete disagreement]
    -0.18627450980392224

**Comment by soumendra** [comment]
That is not what complete disagreement means for cohen's *kappa*.

*kappa* doesn't look at disagreement in the individual ratings, but it looks at the disagreement in various levels/classes.

In your example, a rating disagreement for a rated item (class 0 vs class 2 for the first rated item, class 3 vs class 1 in the second rated item, etc) is not used in calculating *kappa*. Instead, it'll look at how many ratings are there in class 0, class 1, class 2, etc for each of the raters (x and y), and then it'll calculate *kappa* from that.

Note: ml-metrics's deduction of min_rating and max_rating is wrong (in *kappa()*). So you should supply that yourself if you are using the function. scikit-learn's implementation is better written - https://github.com/scikit-learn/scikit-learn/blob/14031f6/sklearn/metrics/classification.py#L278 .

```python
import numpy as np
from ml_metrics import kappa
from sklearn.metrics import cohen_kappa_score

x = np.array([0,3,2,4,0,2,0,4,3,0,2])  
y = np.array([2,1,3,2,3,4,2,1,4,3,1])

a = kappa(x, y, min_rating = 0, max_rating = 4)
b = kappa(x, y)
c = cohen_kappa_score(x,y)

print(a)
print(b)
print(c)
```

**Comment by prakashjayy** [comment]
Thanks. I am closing the issue
